pkg/dbop/status_test.go

```diff
@@ -2,6 +2,7 @@ package dbop
 
 import (
 	"context"
+	"github.com/cybozu-go/moco/pkg/constants"
 
 	mocov1beta2 "github.com/cybozu-go/moco/api/v1beta2"
```

Comment:
The format does not look right.
```suggestion

	"github.com/cybozu-go/moco/pkg/constants"
	mocov1beta2 "github.com/cybozu-go/moco/api/v1beta2"
```